docs/architecture/cloud-native/azure-development-stack.md

```diff
@@ -1,141 +1,180 @@
 ---
-title: The Azure Cloud Native Development Stack
-description: Architecting Cloud Native .NET Apps for Azure | The Azure Cloud Native Development Stack
-author: ardalis
-ms.date: 06/30/2019
+title: The Azure cloud-native development stack
+description: Architecting Cloud Native .NET Apps for Azure | The Azure cloud-native development stack
```

Comment:
Please take a look at my comment re. descriptions here
https://github.com/dotnet/docs/pull/13709#discussion_r312216653

Reply:
From https://www.orbitmedia.com/blog/seo-best-practices/

2. Meta descriptions
Meta descriptions, while not as important in search engine rankings, are extremely important in getting users to click through from the search engine results page (SERP) to your website. Meta descriptions should use keywords wisely, but more importantly they should include a compelling description on which a user would want to click on.

Much like title tags, the SERPs will highlight keywords that the user searched for, increasing the likelihood of the user clicking through to your website.

Reply:
Corrected